index.js

```diff
@@ -86,7 +86,7 @@ const processor = (report, reporterOptions = {}) => {
 
   const launchObj = rpClient.startLaunch({
     name: process.env.RP_LAUNCH_NAME || options.launchname || 'Unit Tests',
-    tags: options.tags,
+    tags: process.env.RP_TAGS ? process.env.RP_TAGS.split(",") : options.tags,
```

Comment:
@leandroeo Would you mind moving all this options(token\launchname\project\tags) selection logic to `('./utils/getOptions').getOptions()`?; I think it will be realy nice to handle options in one place...

Reply:
At least it will make reporter code cleaner

Reply:
@leandroeo up^

Reply:
Hi guys, I can work on that in 2 or 3 days.

Reply:
Hey @BorisOsipov, I just moved all the logic to the getOptions, can you have a look?

Reply:
@leandroeo yeah looks good. Seems now need to fix merge conflict.

Reply:
Sorry, looks like the guys already added support for it and redesigned the class.
I'm closing this pull request.